chemprop/train/loss_functions.py

```diff
@@ -87,16 +104,29 @@ def mcc_class_loss(predictions: torch.tensor, targets: torch.tensor, data_weight
     # shape(batch, tasks)
     # (TP*TN-FP*FN)/sqrt((TP+FP)*(TP+FN)*(TN+FP)*(TN+FN))
     torch_device = predictions.device
-    TP = torch.sum(targets * predictions * data_weights * mask, axis = 0).to(torch_device)
-    FP = torch.sum((1 - targets) * predictions * data_weights * mask, axis = 0).to(torch_device)
-    FN = torch.sum(targets * (1 - predictions) * data_weights * mask, axis = 0).to(torch_device)
-    TN = torch.sum((1 - targets) * (1 - predictions) * data_weights * mask, axis = 0).to(torch_device)
-    loss = 1 - ((TP*TN-FP*FN)/torch.sqrt((TP+FP)*(TP+FN)*(TN+FP)*(TN+FN)))
+    TP = torch.sum(targets * predictions * data_weights * mask, axis=0).to(torch_device)
```

Comment:
this suggestion also applies to all lines up to L119. You don't need a `.to()` call if all of your input tensors are on the proper device because the output tensor is created on the same device
```suggestion
    TP = (targets * predictions * data_weights * mask.sum(axis=0)
```

Reply:
Going through and removing the superfluous .to() calls.

What's the motivation to change from `torch.sum(..., axis=0)` to `(...).sum(axis=0)`. They look equivalent to me. In my opinion the former is easier to read because you don't call class functions from a ")".